### PR TITLE
Campaign priority description

### PR DESCRIPTION
**Pull request by patrykgruszka**
Docs for the https://github.com/mautic/mautic/pull/12721

<a href="https://gitpod.io/#https://github.com/mautic/user-documentation/pull/200"></a>

### CODE REVIEW

**Comment by fakela** [comment]
This looks good to me @RCheesley please take a look

File: docs/campaigns/creating_campaigns.rst [hunk 1]
```diff
@@ -50,6 +50,7 @@ To begin creating Campaigns, perform the following steps:
 #. Optionally, you can set the following properties:
 
 -  **Category** - Choose a Category to assign your Campaign to. Categories help you organize your Campaigns. To learn more about creating and managing Categories, see :doc:`/categories/categories-overview`.
+-  **Priority** - Choose a priority for your campaign. Campaigns with higher priority will be processed before campaigns with normal priority.
```
Comment (RCheesley) [inline docs/campaigns/creating_campaigns.rst:53]:
```suggestion
-  **Priority** - Choose a priority for your Campaign. Campaigns with higher priority will be processed before Campaigns with normal priority.
```

Vale fixes

**Review by RCheesley — CHANGES_REQUESTED** [review]
Just a couple of small vale fixes.

**Review by RCheesley — CHANGES_REQUESTED** [review]
Just one thing flagged up now that Reviewdog/Vale is working again then we are good to go!

Just one question @patrykgruszka - is this something being merged into Mautic 5, or is it in 4 as well? Main is for 4.x and the 5.x branch is for 5.0 so it may require a rebase.

**Comment by RCheesley** [comment]
@patrykgruszka this one needs rebasing please, to the 5.x branch! Thanks!

**Comment by patrykgruszka** [comment]
This change will not be implemented. I am closing this PR

**Comment by RCheesley** [comment]
OK thanks!

**Pull request closed without merging**